seeds/01-instructions.js

```diff
@@ -0,0 +1,13 @@
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('instructions')
+    .truncate()
+    .then(function() {
+      // Inserts seed entries
+      return knex('instructions').insert([
+        { recipe_id: 1, steps: 'Put the dough in the oven' },
+        { recipe_id: 1, steps: 'Smear the tomatoes on the dough' },
```

Comment:
I don't think I've ever heard `smear` when used w/ a recipe before 😂 Maybe that's why your next step is uber eats.